src/graph/graph.c

```diff
@@ -182,34 +182,31 @@ static inline size_t _Graph_EdgeCap(const Graph *g) {
 	return g->edges->itemCap;
 }
 
-// Locates edges connecting src to destination.
-void _Graph_GetEdgesConnectingNodes(const Graph *g, NodeID src, NodeID dest, int r, Edge **edges) {
-	ASSERT(g && src < Graph_RequiredMatrixDim(g) && dest < Graph_RequiredMatrixDim(g) &&
-		   r < Graph_RelationTypeCount(g));
-
+static void _CollectEdgesFromEntry
+(
+	const Graph *g,
+	NodeID src,
+	NodeID dest,
+	int r,
+	EdgeID edgeId,
+	Edge **edges
+) {
 	Edge e;
-	EdgeID edgeId;
-	e.relationID = r;
-	e.srcNodeID = src;
-	e.destNodeID = dest;
 
-	// relation map, maps (src, dest, r) to edge IDs.
-	GrB_Matrix relation = Graph_GetRelationMatrix(g, r);
-	GrB_Info res = GrB_Matrix_extractElement_UINT64(&edgeId, relation, src, dest);
-
-	// No entry at [dest, src], src is not connected to dest with relation R.
-	if(res == GrB_NO_VALUE) return;
+	e.relationID  =  r;
+	e.srcNodeID   =  src;
+	e.destNodeID  =  dest;
 
 	if(SINGLE_EDGE(edgeId)) {
-		// Discard most significate bit.
-		edgeId = SINGLE_EDGE_ID(edgeId);
-		e.entity = DataBlock_GetItem(g->edges, edgeId);
-		e.id = edgeId;
+		// discard most significate bit
+		edgeId    =  SINGLE_EDGE_ID(edgeId);
+		e.id      =  edgeId;
+		e.entity  =  DataBlock_GetItem(g->edges,  edgeId);
 		ASSERT(e.entity);
 		array_append(*edges, e);
 	} else {
-		/* Multiple edges connecting src to dest,
-		 * entry is a pointer to an array of edge IDs. */
+		// multiple edges connecting src to dest,
+		// entry is a pointer to an array of edge IDs
 		EdgeID *edgeIds = (EdgeID *)edgeId;
```

Comment:
MSB issue

Reply:
Correct, on the branch you've introduce recently doesn't applies to `master` just yet